woocommerce/woocommerce-ios
# [Woo POS] M2: Animate transition between 'Ready for Payment' and 'Payment Processing' states

Part of: #13444 


## Description


When transitioning between 'Ready for Payment' and 'Payment Processing' states:
1. TotalsView becomes full screen and changes color (https://github.com/woocommerce/woocommerce-ios/pull/13462)
2. Cart view closes with animation (https://github.com/woocommerce/woocommerce-ios/pull/13463)
4. 'Ready for payment' message view seamlessly transitions to 'Payment Processing' message view
5. 'Payment Processing' message view centers after the delay

⚠️ P.S removed the solution for total fields animating diagonally when checkout is opened for a second time (https://github.com/woocommerce/woocommerce-ios/pull/13463#pullrequestreview-2210720796). The solution didn't prove to be sound.

## Testing information


1. Open Woo -> Menu -> Point of Sale Mode
2. Add items to cart
3. Check out
4. Confirm Ready for Payment -> Payment Processing views transition with animation as in designs p91TBi-bHy-p2

## Screenshots


https://github.com/user-attachments/assets/10fe8928-de1a-4f96-b24a-828364d36589


---
